site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -1681,11 +1681,15 @@ public function getBuildStatusOfGradeable(string $gradeable_id): void {
                 $logs = $this->getBuildLogs($gradeable_id);
 
                 $needle = 'The submitty configuration validator detected the above error in your config.';
+                $makeErrorNeedle = 'MAKE ERROR';
```

Comment:
can we use more common term? maybe just error
because this will only catch only when it finds "MAKE ERROR", won't work when any other "error" is returned.